internal/statemachine/classic_states.go

```diff
@@ -1255,6 +1259,29 @@ func (stateMachine *StateMachine) populateClassicRootfsContents() error {
 	return nil
 }
 
+// Set a default locale if one is not configured beforehand by other customizations
+func (stateMachine *StateMachine) setDefaultLocale() error {
+	classicStateMachine := stateMachine.parent.(*ClassicStateMachine)
+
+	defaultPath := filepath.Join(classicStateMachine.tempDirs.chroot, "etc", "default")
+	localePath := filepath.Join(defaultPath, "locale")
+	localeBytes, err := osReadFile(localePath)
+	if err == nil && strings.Contains(string(localeBytes), "LANG=") {
+		return nil
+	}
+
+	err = osMkdirAll(defaultPath, 0755)
```

Comment:
etc/default is shipped by base-files so i don't think this is necessary but it's also not harmful i guess.